src/openrct2/scripting/ScEntity.hpp

```diff
@@ -56,8 +58,15 @@ namespace OpenRCT2::Scripting
                     case EntityType::Vehicle:
                         return "car";
                     case EntityType::Guest:
+                        if (targetApiVersion == 33)
```

Comment:
Probably better to use `<=` in case someone (accidentally) specifies a lower API version.

Reply:
But that would not specifically target a version then? If they mistarget it they will probably see errors anyway.

Reply:
It used to be `peep` up until API > 33 anyway, so even if the specified version is < 33, it's technically correct. Targeting a version before 33 should definitely give the player an error.